conf/local.conf.sample

```diff
@@ -169,6 +169,8 @@ BB_DISKMON_DIRS ??= "\
 #file://.* http://someserver.tld/share/sstate/PATH;downloadfilename=PATH \n \
 #file://.* file:///some/local/dir/sstate/PATH"
 
+# License whitelist to include FFMPEG in the extras feed like other distributions.
+LICENSE_FLAGS_WHITELIST += "commercial"
```

Comment:
Enabling `commercial` licenses like this would whitelist all recipes with commercial licenses - which we definitely don't want.

Does something like `LICENSE_FLAGS_WHITELIST = "commercial_ffmpeg"` work instead, [per the manual](https://docs.yoctoproject.org/3.1.19/dev-manual/dev-manual-common-tasks.html#enabling-commercially-licensed-recipes)?

Reply:
Changed